portality/tasks/journal_in_out_doaj.py

```diff
@@ -50,10 +60,10 @@ def cleanup(self):
         # remove the lock on the journal
         job = self.background_job
         params = job.params
-        journal_id = params.get("set_in_doaj__journal_id")
+        journal_ids = params.get("set_in_doaj__journal_ids")
```

Comment:
cls.get_param(params, 'journal_ids'). Maybe with a default of [], not sure if get_param supports defaults, but can be easily put in.